src/main/java/net/coderbot/terrestria/init/TerrestriaBiomes.java

```diff
@@ -177,6 +341,25 @@ public static void init() {
 				.addSpawnEntry(new Biome.SpawnEntry(EntityType.WOLF, 5, 4, 4))
 				.build());
 
+		WOODED_CYPRESS_HILLS = register("wooded_cypress_hills", TerrestriaBiome.builder()
+				.configuredSurfaceBuilder(SurfaceBuilder.DEFAULT, SurfaceBuilder.GRASS_CONFIG)
+				.precipitation(Biome.Precipitation.RAIN).category(Biome.Category.FOREST)
+				.depth(0.45F)
+				.scale(0.3F)
+				.temperature(0.7F)
+				.downfall(0.8F)
+				.waterColor(4159204)
+				.waterFogColor(329011)
+				.addDefaultFeatures(LAND_CARVERS, STRUCTURES, LAKES, DUNGEONS, FOREST_FLOWERS, MINEABLES, ORES, DISKS,
+						DEFAULT_FLOWERS, DEFAULT_MUSHROOMS, FOREST_GRASS, DEFAULT_VEGETATION, SPRINGS, FROZEN_TOP_LAYER)
+				.addTreeFeature(TerrestriaFeatures.CYPRESS_TREE, 9)
+				.addTreeFeature(TerrestriaFeatures.TALLER_BIRCH_TREE, 4)
+				.addStructureFeature(Feature.STRONGHOLD)
+				.addStructureFeature(Feature.MINESHAFT, new MineshaftFeatureConfig(0.004D, MineshaftFeature.Type.NORMAL))
+				.addDefaultSpawnEntries()
+				.addSpawnEntry(new Biome.SpawnEntry(EntityType.WOLF, 5, 4, 4))
+				.build());
+
 		CYPRESS_SWAMP = register("cypress_swamp", TerrestriaBiome.builder()
```

Comment:
this biome should need some grass color changes and maybe some denser ground cover